app/uk/gov/hmrc/eusubsidycompliance/controllers/UndertakingController.scala

```diff
@@ -55,20 +55,21 @@ class UndertakingController @Inject() (
   }
 
   def create: Action[JsValue] = authenticator.authorisedWithJson(parse.json) { implicit request => _ =>
-    withJsonBody[UndertakingCreate] { undertaking: UndertakingCreate =>
+    withJsonBody[UndertakingCreate] { undertakingCreate: UndertakingCreate =>
```

Comment:
Just renamed to the type as Undertaking has few concepts and makes reading easier